components/empty-state/empty-state-simple.js

```diff
@@ -18,20 +18,34 @@ class EmptyStateSimple extends PropertyRequiredMixin(RtlMixin(LitElement)) {
 			 * @type {string}
 			 */
 			description: { type: String, required: true },
+			_hasDescription: { type: Boolean, reflect: true, attribute: '_has-description' }
 		};
 	}
 
 	static get styles() {
-		return [bodyCompactStyles, emptyStateStyles, emptyStateSimpleStyles];
+		return [bodyCompactStyles, emptyStateStyles, emptyStateSimpleStyles, css`
+			:host([_has-description]) .empty-state-container {
+				align-items: center;
+				column-gap: 0.5rem;
+				display: flex;
+				flex-wrap: wrap;
+			}
+		`];
 	}
 
 	render() {
 		return html`
-			<p class="d2l-body-compact d2l-empty-state-description">${this.description}</p>
-			<slot class="action-slot"></slot>
+			<div class="empty-state-container">
+				<p class="d2l-body-compact d2l-empty-state-description">${this.description}</p>
+				<slot class="action-slot" @slotchange="${this._handleActionSlotChange}"></slot>
+			</div>
 		`;
 	}
 
+	_handleActionSlotChange() {
+		this._hasDescription = this.description.trim().length > 0;
```

Comment:
Were you able to get the vdiffs running locally? It looks like the ones on the PR weren't running because if `this.description` is undefined then we can't use `trim`.

It's actually possible to do this all in the css though by checking if `description` itself is defined and then you can avoid an extra property and slotchange listener:
```
:host([description]) .empty-state-container
```
which could live in the styles file